static/app/views/performance/browser/webVitals/pageSamplePerformanceTable.tsx

```diff
@@ -69,23 +73,23 @@ export function PageSamplePerformanceTable({transaction}: Props) {
   // We can't query by performance score yet, so we're using LCP as a best estimate
   const {data: goodData, isLoading: isGoodTransactionWebVitalsQueryLoading} =
     useTransactionSamplesWebVitalsQuery({
-      limit: 3,
+      limit: limit / 3,
```

Comment:
m: there's a risk of this being decimal if someone provides a custom limit that isn't divisible by 3. Should we `Math.round`?